### DISCUSSION TITLE
Please support MALI GPU

### DISCUSSION THREAD

**Discussion by castlenova69** [skyline-emu/skyline#1948] [Game Requests]
MALI GPU

**Comment by AndresNorm** [comment]
They are supporting Mali, atleast on Edge build.
Just wait its coming 1week from now or 2weeks

**Reply by Loganavter** [reply]
> They are supporting Mali, atleast on Edge build. Just wait its coming 1week from now or 2weeks

Are you talking about a particular GPU or all the GPUs Mali?

**Reply by JBapt23** [reply]
> > They are supporting Mali, atleast on Edge build. Just wait its coming 1week from now or 2weeks
> 
> Are you talking about a particular GPU or all the GPUs Mali?

Hi; sorry for disturb; maybe minor Mali's, 'cause i have an: 

Helio G96 (Mediatek Gaming Processor, Mediatek says that, not me) 
Mali-G57 MC2 
8GB RAM 
128GB Rom 

Edge shows "Your GPU is not supported" or in spanish something like "Your GPU don't support custom GPU drivers", also i've tryed Edge 19 & 22 when i select the game (Let's Go Pikachu) the screen stucks black, 

Also i've tryed Edge 9 (this www.youtube.com/watch?v=FNeNERCOOok&t=8s works for Exynos) and have the same result

Thanks for answer

**Reply by JERD001** [reply]
Short answer: no drivers for Mali
Long answer: Mali doesn't have drivers available to change it, therefore the option is blocked. Don't follow videos that give you false hopes for a improvement for Mali GPUs, if it's not available is because of something. If you wish better performance and interchangeable drivers your only option rn is buying a phone with snapdragon.

**Reply by JBapt23** [reply]
Thanks for answer.

Ok, maybe Mali don't have Custom Drivers today, and i gone wrong but, what's about Vulkan support like mine and others? (if the problem its by hardware i understand); afaik the Mali G78 runs skyline, i'm gonna wait but in some way it's like the developers leaving us appart.

I'm testing free 3 day Egg and it's buggy but it's running that's proof that Mali GPU's can run the games not only SnapDragon's SoC (i'm not comparing 'cause Sky team respect the Open Source rules and the rivals have so noisy and bad romours).

**Reply by JERD001** [reply]
Mali has had Vulkan support for a long time, really long time. You can see people with Mali G72 and even Mali G51 running Skyline although not greatly. You can run games, only GPU that can't are PowerVR since they are extremely old and low end at this point. With Mali you can run games but you'll have a worse experience than with equivalent Adreno, not because of drivers but Mali is slower than Adreno so expect lower performance.
